src/lib/console.ts

```diff
@@ -4,24 +4,6 @@
  */
 import type { EventData } from './MessageManager'
 
-export const injectedJavaScript = `
-if (!globalThis._console) {
-  globalThis._console = { ...globalThis.console };
-  globalThis.console = new Proxy(globalThis.console, {
-    get(target, property) {
-      return (...args) => {
-        globalThis._console[property](...args)
-        try {
-          window.ReactNativeWebView.postMessage(JSON.stringify({cmd: 'console', fn: property, data: args.join(' ') }))
-        } catch (e) {
-          globalThis._console[property]('[REACT-NATIVE-WEBVIEW] ', e)
-        }
-      }
-    }
-  });
-}
-`
-
```

Comment:
That was the culprit of both https://shapeshift-dao.sentry.io/issues/253532, https://github.com/shapeshift/web/issues/6766 and probably a bunch of other bugs we're not aware of